go-controller/pkg/util/multi_network.go

```diff
@@ -432,6 +432,10 @@ func newLayer2NetConfInfo(netconf *ovncnitypes.NetConf) (NetInfo, error) {
 		return nil, fmt.Errorf("invalid %s netconf %s: %v", netconf.Topology, netconf.Name, err)
 	}
 
+	if netconf.Role == types.NetworkRolePrimary {
+		excludes = primaryNetworkExcludedIPs(subnets)
```

Comment:
This would override users configuring the NAD exclude subnet field.

Reply:
I know. I assumed we don't want users to use that for user defined primary nets. 

... do we ? @tssurya

Reply:
hmm good question I think we should continue to support excludesubnets just to stay consistent with user expectations when using a NAD. maybe we should append to user input if any.